Client/Frontend/Browser/TabManager.swift

```diff
@@ -138,6 +143,10 @@ class TabManager : NSObject {
             request = defaultNewTabRequest
         }
 
+        if (configuration != nil) {
+            configuration.preferences.javaScriptCanOpenWindowsAutomatically = !(prefs.boolForKey("blockPopups") ?? true)
```

Comment:
The Swifty way to do this would be `configuration?.preferences.javaScriptCanOpenWindowsAutomatically = ...`. That way, we can get rid of the outer `if` statement.

While here, I wonder why `request` and `configuration` are implicitly unwrapped optionals instead of regular optionals. Maybe change them here so this can be safer?